tests/test_chromatic.py

```diff
@@ -1437,7 +1437,10 @@ def __repr__(self):
         parallactic_angle=0.*galsim.degrees)
     chrom_shear = lambda w: galsim.Shear(g1=0.2+0.2*(w-500.)/500.,g2=0.) if w<1000. else \
         galsim.Shear(g1=0.4, g2=0.)
-    chrom_shift_y = lambda w: scale*(w-500.)
+    # The 0.5 in the line below used to be 1.0.  When we improved the .thin() method, this test
+    # started to fail in 0.1% of pixels.  Making the chromaticity less extreme allowed the test
+    # to pass.
+    chrom_shift_y = lambda w: 0.5*scale*(w-500.)
```

Comment:
There's a new file tests/chromatic_reference_images/r_exact_Airy.fits in this PR.  Should that have been used here somewhere?